tests/unit/tests_high_five_base.cpp

```diff
@@ -242,8 +242,13 @@ BOOST_AUTO_TEST_CASE(HighFiveGroupAndDataSet) {
                           DataSetException);
 
         // here we use the other signature
-        DataSet dataset_chuncked = file.createDataSet<float>(
+        DataSet dataset_chunked = file.createDataSet<float>(
             CHUNKED_DATASET_NAME, dataspace, goodChunking);
+
+        // Try resize out of bounds
+        BOOST_CHECK_THROW(dataset_chunked.resize({5, 7}), DataSetException);
+        // Try resize invalid dimensions
+        BOOST_CHECK_THROW(dataset_chunked.resize({1, 2, 3}), DataSetException);
```

Comment:
This is not what I meant. I meant to to try a bad chunking with more dimensions that the dataset. You've already provided a resizing test elsewhere.

Reply:
I see. Maybe also test a dataset resize that is smaller than the chunking size?

Reply:
That wouldn't hurt for sure.